src/main/java/com/hsleiden/vdlelie/services/EmailService.java

```diff
@@ -0,0 +1,97 @@
+package com.hsleiden.vdlelie.services;
+
+import com.hsleiden.vdlelie.model.Account;
+import jakarta.mail.MessagingException;
+import jakarta.mail.internet.MimeMessage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.mail.javamail.JavaMailSender;
+import org.springframework.mail.javamail.JavaMailSenderImpl;
+import org.springframework.mail.javamail.MimeMessageHelper;
+import org.springframework.scheduling.annotation.Async;
+import org.springframework.stereotype.Service;
+import org.thymeleaf.TemplateEngine;
+import org.thymeleaf.context.Context;
+
+import java.util.ArrayList;
+import java.util.Properties;
+import java.util.List;
+import java.util.Arrays;
+
+@Service
+public class EmailService {
+
+    JavaMailSender mailSender = getJavaMailSender();
+    private final TemplateEngine templateEngine;
+    private final AccountServiceImpl accountService;
+
+
+    @Autowired
+    public EmailService(TemplateEngine templateEngine, AccountServiceImpl accountService){
+        this.templateEngine = templateEngine;
+        this.accountService = accountService;
+    }
+
+    @Bean
+    public JavaMailSender getJavaMailSender() {
+        JavaMailSenderImpl mailSender = new JavaMailSenderImpl();
+        mailSender.setHost("smtp.gmail.com");
+        mailSender.setPort(587);
+
+        mailSender.setUsername("draakje11111@gmail.com");
+        mailSender.setPassword("jkww pvwu suku pdoe ");
```

Comment:
Normaal password?

Reply:
random generated api password dus je kan niet zomaar mijn email in alleen beetje kut dat de repo open is maarja ik retract hem wel als wij klaar zijn met de opdracht